README.md

```diff
@@ -11,15 +11,16 @@
 
 A link aggregator / reddit clone for the fediverse.
 
-[Lemmy Dev instance](https://dev.lemmy.ml) *for testing purposes only*
+[Lemmy Dev instance](https://dev.lemmy.ml) _for testing purposes only_
 
 This is a **very early beta version**, and a lot of features are currently broken or in active development, such as federation.
- 
-Front Page|Post
----|---
-![main screen](https://i.imgur.com/y64BtXC.png)|![chat screen](https://i.imgur.com/vsOr87q.png)
+
+| Front Page                                      | Post                                            |
+| ----------------------------------------------- | ----------------------------------------------- |
```

Comment:
I'm not a fan of these auto-width fillers, I'm guessing they were inserted by something automatically. We all don't use editors that have that same plugin so simpler is better.

Reply:
Sure, I'll remove them. These have been automatically added by my formatter so kind of just them.